manifests/check.pp

```diff
@@ -49,6 +49,7 @@
   $package_name = 'monit',
   $service_name = 'monit',
 ) {
+  include monit::params
```

Comment:
This is the only line I'm not sure you'll be happy with. Is this how you want to expand monit::params into the type?